### PR TITLE
[near] cache getValidator and getCommission

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

- Everytime a user go on their account they load all the Near nodes and the commissions for these nodes. We now add a cache on these function to only call these functions if the data is not in the cache

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-10172

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/families/near/api/archive-node-sdk.ts [hunk 1]
```diff
@@ -248,46 +249,54 @@ export const getStakingPositions = async (
...
-  }
+      return Math.round((parsedResult.numerator / parsedResult.denominator) * 100);
+    }
 
-  return null;
-};
+    return null;
+  },
+  () => "",
+  { ttl: 10 * 60 * 1000 },
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/near/api/archive-node-sdk.ts:301]: Unimportant, but I think the TTL can be 30min for both caches

**Review by haammar-ledger — COMMENTED** [review]
Need a changeset + typo in the PR's title :slightly_smiling_face:

**Comment by Wouzz** [comment]
@lvndry Wrong JIRA ticket linked to this PR :eyes:

**Comment by haammar-ledger** [comment]
@lvndry Wrong JIRA ticket linked to this PR 👀

**Review by haammar-ledger — APPROVED** [review]
:raised_hands:

**Pull request merged**